front-end/src/Components/UserTask.jsx

```diff
@@ -1,12 +1,14 @@
+/* eslint-disable react/prop-types,react/destructuring-assignment,no-unused-vars,react/jsx-no-bind */
```

Comment:
cheater? ))
Agree that some eslint objections can be reduced in priority. Some of them even can be disabled, but try to not to do it for one file.

Reply:
Yeah, i temporarily disabled these rules just to see if code works. Forgot to fix them.